lookup.js

```diff
@@ -87,20 +87,7 @@ exports.init = function (sbot, config) {
     details.feedformat = validate.detectFeedFormat(content.subfeed)
     details.feedpurpose = content.feedpurpose || details.feedpurpose
     details.metafeed = content.metafeed || details.metafeed
-    details.metadata = {} || details.metafeed
```

Comment:
this look like another bug 

```diff
- {} || details.metafeed
+ {} || details.metadata
```

please check, I don't know what I'm doing

Reply:
I don't think this is correct. The metadata is added to the message content as a flat structure (as keys). Not as an object, this confusing part here is maybe that you pass in an object to the findOrCreate.